### PR TITLE
add copy for landing pages

### PR DESCRIPTION
**Pull request by RickardssonJ**

### CODE REVIEW

File: src/components/LearningMaterial/index.tsx [hunk 1]
```diff
@@ -87,12 +87,18 @@ export default function LearningMaterial({
     }))
     return [
       ...getUnorderedListAccordion(
-        'Learning outcomes',
+        `${landingPageCopy?.LearningOutcomes}`,
```
Comment (tasola) [inline src/components/LearningMaterial/index.tsx:90]: landingPageCopy doesn't need to be optional anymore, right?

File: src/components/MetadataContainer/MetadataContainer.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ export default function MetadataContainer({
         {!!level?.data && (
```
Comment (tasola) [inline src/components/MetadataContainer/MetadataContainer.tsx:88]: Same in this file: landingPageCopy doesn't need to be optional anymore, right?

Comment (tasola) [reply src/components/MetadataContainer/MetadataContainer.tsx:88]: There's lots of unused imports in this file as well, please remove them

**Review by tasola — CHANGES_REQUESTED** [review]
Small suggestions

File: src/components/LearningMaterial/index.tsx (hunk 1 above)
Comment (RickardssonJ) [reply src/components/LearningMaterial/index.tsx:90]: No you are right. They are required in the CMS so there is no need for optional anymore.

File: src/components/MetadataContainer/MetadataContainer.tsx (hunk 2 above)
Comment (RickardssonJ) [reply src/components/MetadataContainer/MetadataContainer.tsx:88]: I will remove them

**Pull request merged**